scripts/provisioning/s3_setup

```diff
@@ -539,6 +548,15 @@ then
 	create_recovery_tool_account
 fi
 
+# Clean up backgrounddelete account
+if [[ $cleanup_both == true ]]
```

Comment:
Remove the below - 'if [[ $cleanup_dependencies == true ]]' section if its already handled here. Anyways it does not get called now I believe.